src/server/views/prototypes/site-review/assessment/activity/index.njk

```diff
@@ -72,14 +72,22 @@
             {% if viewData.activity.errors[0]['activityDropdown'] %}
               <span class="error-message">{{ viewData.activity.errors[0]['activityDropdown'] }}</span>
             {% endif %}
-            <select class="form-control" id="reinspection-options" name="reinspection-options">
+            <div>
+              <select class="form-control" id="reinspection-options" name="reinspection-options">
               <option value="0">Please select</option>
               <option value="1" {{'selected' if viewData.activity.formData.reason == 1}}>It wasn't necessary</option>
               <option value="2" {{'selected' if viewData.activity.formData.reason == 2}}>Blocked by pseudo customer</option>
               <option value="3" {{'selected' if viewData.activity.formData.reason == 3}}>Blocked by vehicle owner</option>
               <option value="4" {{'selected' if viewData.activity.formData.reason == 4}}>I felt under threat from the garage staff</option>
               <option value="5" {{'selected' if viewData.activity.formData.reason == 5}}>Other</option>
             </select>
+            </div>
+            <div style="margin-top: 10px;">
```

Comment:
Only mention it because this is a view we've refactored, please use class .u-margin-top-10

Reply:
OK - no probs Raf. Agreed.